homeassistant/components/aemet/__init__.py

```diff
@@ -26,6 +26,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: AemetConfigEntry) -> boo
     latitude = entry.data[CONF_LATITUDE]
     longitude = entry.data[CONF_LONGITUDE]
     update_features: int = UpdateFeature.FORECAST
+    if entry.options.get(CONF_RADAR_UPDATES, False):
+        update_features |= UpdateFeature.RADAR
```

Comment:
Oh and last question, why is this optional? Any specific reason?

Reply:
> Oh and last question, why is this optional? Any specific reason?

This adds additional API calls that may cause coordinator updates failures and since the AEMET OpenData API is extremely unstable lately I preferred to make it optional and let users decide if they want to enable it or not, just like the weather station data.